wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/transformers/DynamicParameterProvider.java

```diff
@@ -48,6 +48,11 @@ private ClousureCall(Closure closure) {
     public Object call() {
       return closure.call();
     }
+
+    @Override
+    public String toString(){
+      return Closure.class.getName();
```

Comment:
Are you sure here? Above you just replaced closure to dynamic provider

Reply:
as we support groovy Closure and java Supplier

Reply:
I mean, toStriing impl